scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1499,9 +1668,9 @@
         (is (= (:normal expected-pool->pending-jobs)
                (:normal (sched/remove-matched-jobs-from-pending-jobs pool->pending-jobs (:normal pool->matched-job-uuids) :normal))))))))
 
-(deftest test-handle-resource-offers
+(deftest test-handle-resource-offers-mesos
```

Comment:
Same as above... Can this be refactored to share more code with the next test by having the deftest call another function with the (is ...) tests?